test/environment.py

```diff
@@ -32,6 +32,7 @@ def encode_bytes(d):
                 encode_bytes(d[i])
             else:
                 if isinstance(d[i], bytes):
+                    assert False, "Zeph thinks this code is unreachable"
```

Comment:
pylance was complaining that `v` isn't defined... and I agree

Reply:
The next line should likely be `d[i] = base64.b64encode(d[i]).decode()`. Can you replace it with that?